pages/linux/synoupgrade.md

```diff
@@ -0,0 +1,36 @@
+# synoupgrade
+
+> Upgrade a Synology DiskStation Manager (DSM) from the command-line.
+> More information: <https://www.synology.com/dsm>.
+
+- Check upgrades printing the result:
+
+`sudo synoupgrade --check`
+
+- Check for patches without upgrading the DSM version:
+
+`sudo synoupgrade --check-smallupdate`
+
+- Download latest upgrade available (use `--download-smallupdate` for patches):
+
+`sudo synoupgrade --download`
+
+- Start the upgrade process:
+
+`sudo synoupgrade --start`
+
+- Upgrade to the latest version automatically:
+
+`sudo synoupgrade --auto`
+
+- Apply patches without upgrading the DSM version automatically:
+
+`sudo synoupgrade --auto-smallupdate`
+
+- Upgrade the DSM using a patch file (should be an absolute path):
+
+`sudo synoupgrade --patch {{path/to/file.pat}}`
```

Comment:
If the path should be absolute, why not indicate it in the example?
```suggestion
`sudo synoupgrade --patch {{/path/to/file.pat}}`
```

Reply:
Based on the `ksvgtopng5` page, we don't start with `/` the absolute paths.

Reply:
I just saw that this is indeed not in our guidelines, but we use that everywhere else. (in 153 pages). It's a very good indicator that shows that a path should be absolute.

Reply:
@navarroaxel: This is indeed something that we do by convention. Absolute paths are prefixed with a forward slash `/`.